File: Selenium/Project/CRM_Activity3.java
package demosuit;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeTest;
import org.testng.annotations.Test;

public class CRM_Activity3 {
	WebDriver driver;

	@BeforeTest
	public void beforeMethod() {
		// Create a new instance of the Chrome driver
		driver = new ChromeDriver();
		// Open the URL
		driver.get("http://alchemy.hguy.co/crm");

	}
	
	 @Test
	public void getCopyrightText() {
		

		// Find the first copyright text element (typically in the footer)
		WebElement copyrightText = driver.findElement(By.id("admin_options"));

		// Print the copyright text to the console
		
		System.out.println("First copyright text: " + copyrightText.getText());
		Assert.assertEquals(copyrightText.getText(), "© Supercharged by SuiteCRM");
	}

	// Tear down function
	@AfterClass
	public void tearDown() {
		// Close the browser
		driver.quit();
	}
}

File: Selenium/Project/CRMActivity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:
    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/crm")
    footer_copyright_element = driver.find_element(By.ID, "admin_options")
    
    # Get the copyright text
    copyright_text = footer_copyright_element.text
    
    # Print the copyright text to the console
    print("Copyright Text:", copyright_text)
    driver.quit()